packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1607,7 +1619,10 @@ export default mixins(
 				await this.$store.dispatch('workflows/setNewWorkflowName');
 				this.$store.commit('setStateDirty', false);
 
-				await this.addNodes([DEFAULT_START_NODE]);
+				// Set default values for the new default node
+				const nodes = this.setDefaultStartNode();
```

Comment:
thanks for this...  I see the problem now that we are updating our constant object.. a simpler fix here might be 
```suggestion
				const nodes = [{...DEFAULT_START_NODE}];
```
and then we can add `disabled: false` to our DEFAULT_START_NODE object

Reply:
this should also prevent this issue from any other aspects of the start node... for example node settings

Reply:
@mutdmour Updated. Yup, the spread operator fixed everything related to updating the props from/to Default Start Node object.